src/main/resources/global/90-init-alfresco.sh

```diff
@@ -15,6 +15,9 @@ CONFIG_FILE=${CONFIG_FILE:-${CATALINA_HOME}'/shared/classes/alfresco-global.prop
 LOG4J_CONFIG_FILE=${LOG4J_CONFIG_FILE:-${CATALINA_HOME}'/webapps/alfresco/WEB-INF/classes/alfresco/extension/dev-log4j.properties'}
 TOMCAT_CONFIG_FILE=${CATALINA_HOME}'/bin/setenv.sh'
 TOMCAT_SERVER_FILE=${CATALINA_HOME}'/conf/server.xml'
+# initialiaze variables for keystores
+DIR_KEYSTORE="${DIR_KEYSTORE:-'/opt/alfresco/keystore'}"
```

Comment:
I think it's best to move these defaults to InitScriptMain as well.
These are not exported here, so they are not available as environment variables in InitScriptMain

Reply:
that's what made the test fail I just realised. I'll move them over as well.